app/views/food/_food.html.erb

```diff
@@ -1,14 +1,9 @@
-<div class="col-xl-4 col-lg-4 col-md-6">
-                    <div class="single_recepie text-center">
-                        <div class="recepie_thumb">
-                            <img src="/img/recepie/recpie_1.png" alt="">
-                        </div>
-                        <h3><%= food.name %></h3>
-                        <span>Appetizer</span>
-                        <p>Time Needs: 30 Mins</p>
-                        <a href="#" class="line_btn">View Full Recipe</a>
-                    </div>
-                </div>
-
 
+    <tr>
+      <%# <th scope="row">1</th> %>
+      <td> <h3><%= food.name %></h3></td>
+      <td><%= food.measuring_unit %></h3></td>
+      <td><h3><%= food.price %></h3></td>
+      <td><h3><%= link_to "Delete", inventory_food_index_path(food_id: food.user_id, id: food.id ), data: {confirm: "Are you sure you want to delete: #{food.name}?",}, method: :delete  %></h3></td>
```

Comment:
Please, update the route to use the good path to destroy food.

Instead of
<img width="1215" alt="Capture d’écran 2022-05-19 à 00 12 23" src="https://user-images.githubusercontent.com/7243132/169163645-b9c72be8-f6d0-4e43-b373-714ab8f75d27.png">

Use 
<img width="1215" alt="Capture d’écran 2022-05-19 à 00 13 32" src="https://user-images.githubusercontent.com/7243132/169163775-8215fe7f-137b-4dd9-81db-cb8063bf4aa4.png">